PayButton/paybutton-server
# [#Issue 556] fix/buttons on payments

Related to #556





Description
---
Don't leak other buttons information on payments.



Test plan
---

1. Create new account
2. Create a button with address: bitcoincash:qzqh7ej2vz26a9xaxq7capzfwgxt5gem9g8rvfxc5t (name it something unique so that you know it's your button)
3. Create a 2nd button with address: bitcoincash:qzqh7rwaq9zm4zcv40lh9c9u50gy07gcesdmja8426 and also name this one uniquely
4. Go back to the 1st button and add this address: ecash:qzqh7ej2vz26a9xaxq7capzfwgxt5gem9g7wczazju
5. Go to the Payments page and see that the button names are correct.
